.env

```diff
@@ -0,0 +1,2 @@
+DB_HOST=mongodb+srv://Evgenii:rP4FXVnf5nRtKlDT@contacts.nwxeew5.mongodb.net/my-contacts?retryWrites=true&w=majority
```

Comment:
.env file must be hidden, add it to .gitignore

Reply:
fixed .gitignore